editoast/src/modelsv2/fixtures.rs

```diff
@@ -0,0 +1,25 @@
+#[cfg(test)]
+pub mod tests {
```

Comment:
You don't need to make a test submodule, just `#[cfg(test)]` its inclusion in the source tree